integreat_cms/release_notes/current/unreleased/2838.yml

```
@@ -0,0 +1,2 @@
+en: Add select all box for machine translations in page form
+de: Auswahlfeld für maschinelle Übersetzungen im Seitenformular hinzufügen
```

```suggestion
de: Füge Auswahlfeld für maschinelle Übersetzungen im Seitenformular hinzu
```


Imperative form is the standard in our release notes, if I remember it correctly... 💭